examples/semantic-cache/cache.py

```diff
@@ -57,7 +57,6 @@ class Cache(SQLModel, table=True):
         sa_column=Column(
             VectorType(768),
             default=None,
```

Comment:
also using `create_vector_index` instead?

Reply:
Vector index syntax is strange, I suggest only show it in orm examples.

Reply:
I guess there might be no high-priority plan of introducing a new syntax in recent versions, because ORM users are already using a good encapsulated one. Even after a new version is introduced I don't think we could avoid using Adaptor in all ORMs.